app/src/main/java/com/isc/hermes/model/Utils/DataAccountOffline.java

```diff
@@ -53,7 +53,11 @@ public void saveDataLoggedAccount(User user) {
         editor.putString(PATH_IMG_US, user.getPathImageUser());
         editor.putString(USER_TYPE, user.getTypeUser());
 
+        System.out.println("user:" + user.getTypeUser());
```

Comment:
Add logging infrastructure methods instead out.println

Reply:
I preferred to delete it, because they were only to be sure of the type of user at a given time.

Reply:
this printout is not necessary at all, it was only to see in which state the user was saved.